pandas/core/indexes/interval.py

```diff
@@ -92,6 +94,18 @@ def _get_interval_closed_bounds(interval):
     return left, right
 
 
+def _maybe_convert_platform_interval(data):
+    """
```

Comment:
can you add a doc-string, you can de-privatize (no leading _)

Reply:
done